### ISSUE TITLE
Inadequate Error Message when sending whole balance.

### ISSUE THREAD

**Issue by dabura667** [hivewallet/hive-js#122]
When user tries to send the whole balance, the error message "Not enough funds" comes up.

A new user who has yet to learn of mining fees will be perplexed. So if send amount = balance, and fee is required, display "This transaction requires a miner's fee. The most you can send after paying the miner's fee is (balance - 0.0001)."

Or you could get fancy and maybe have a "Would you like to send (balance - 0.0001) BTC instead?" with a Yes No selection.

Basically, as long as the user understands why they can't empty their wallet, this would be solved.
